### DISCUSSION TITLE
7月购买了一次pro，结果重装系统没法登录了，只好再买一次pro，结果无法切换到专业版，又买了第三次

### DISCUSSION THREAD

**Discussion by Justlooklookju** [getlantern/lantern-forum-cn#759] [Q&A]
怎样才能登录专业版，而且我想退掉后两次购买
邮箱512383@qq.com

**Comment by whpublic2024** [comment]
他们这是不解决问题了么？

**Reply by Justlooklookju** [reply]
看到回复还以为是蓝灯的，对他们超失望，三年会员换不来一句回复

**Reply by MaxMKMax** [reply]
@Justlooklookju 
您好. 非常抱歉, 您的信息被挤到下方了. 错过了您的信息. 我马上帮您处理!

**Comment by MaxMKMax** [comment]
@Justlooklookju 您提供我 3 各支付单据, 我帮您退掉两个然后其中一个可以给您激活码登入专业版.

**Reply by Justlooklookju** [reply]
![IMG_0816](https://github.com/user-attachments/assets/f27da39a-267a-417f-818e-41df86e95590)
![IMG_0811](https://github.com/user-attachments/assets/842bb14b-6949-4423-8c26-734b60065daf)
第一筆是通過支付寶，第二，三次是信用卡，使用郵箱是512383@qq.com

**Reply by Justlooklookju** [reply]
請問查詢到了嗎

**Reply by Justlooklookju** [reply]
最後詢問一次，請問查詢到我的購買了嗎？
